### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by lpabon**
Part of #44

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 [![Build Status](https://travis-ci.org/kubernetes-csi/driver-registrar.svg?branch=master)](https://travis-ci.org/kubernetes-csi/driver-registrar)
 
+# !NOTE!
+:warning: :warning: DEPRECATED :warning: :warning:
+
+This side car container has been deprecated. See #44 for more information
```
Comment (jsafrane) [inline README.md]: `#44` does not seem to get expanded to link to real #44

Comment (msau42) [reply README.md]: It might be good to mention here that the replacement is to include the CSIDriver object as part of the driver's deployment manifest

**Pull request merged**